gravitee-apim-gateway/gravitee-apim-gateway-policy/src/main/java/io/gravitee/gateway/reactive/policy/PolicyFactoryManager.java

```diff
@@ -52,6 +53,27 @@ public PolicyFactoryManager(Class<? extends PolicyFactory> defaultPolicyFactoryC
         defaultPolicyFactory = extractDefaultPolicyFactory(policyFactories);
     }
 
+    /**
+     * Excludes the {@link DefaultPolicyFactory} if DebugPolicyFactory is present
+     * in the {@link PolicyFactoryManager}, ensuring the correct filter is applied.
+     * <p>
+     * This method prioritizes the {@code debugPolicyFactory} over the {@code defaultPolicyFactory},
+     * providing a more precise filtering mechanism within the {@code PolicyFactoryManager}.
+     * ex- DebugExpressionLanguageConditionFilter over ExpressionLanguageConditionFilter.
+     * </p>
```

Comment:
your comment explains a specific case but it is not an appropriate javadoc.
prefer something like this:
```suggestion
/**
     * Build the manager with a set of {@link PolicyFactory} and select {@param defaultPolicyFactoryClass} as fall back factory, you can exclude a PolicyFactory class thanks to {@param excludedPolicyFactoryClass}
     * @param defaultPolicyFactoryClass the class that will operate as the default policy factory
     * @param excludedPolicyFactoryClass the class to exclude from the {@param policiesFactories}. This one is particularly useful in the case of the Debug Mode, where we want to exclude the {@link DefaultPolicyFactory} from the Spring injected list of {@link PolicyFactory}
     * @param policyFactories
     */
```